.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Could you please exclude all files in `.idea`?

Reply:
These files are only related to formatting (other files are excluded). I could remove them,  but i don't think it is good idea. It took a lot of effort before i managed to get correct formatting settings in place so that checks wouldn't fail and i wouldn't use different format for new/changed lines. Unless You still strongly disagree, I'd leave them there. What did you think?

Reply:
@atsu85 Why don't you use [a style guide](https://github.com/google/styleguide) ?